src/components/MyActivities/pages/Loggedin/MyActivitiesHelp.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import '../../MyActivitiesStyle.scss';
+import help from './help';
+
+function MyActivitiesHelp() {
+  const [helpId, setHelpId] = useState(0);
+
+  return (
+    <>
+      {localStorage.getItem('help-visited') !== 'true' ? (
```

Comment:
If we had help for different screens, the user would see help for one screen I think.